### ISSUE TITLE
Can't login to my Web app

### ISSUE THREAD

**Issue by Hassan2419991** [cypress-io/cypress#28798]
### Current behavior

I have automated the entire backend of my organization, and it was working perfectly fine. However, starting last week, the script is no longer functioning properly. When I enter the email and password and then proceed with the 2FA, instead of logging me in, it redirects me back to the login page. I have attempted to debug the issue through the console, but I haven't been able to identify anything. Please assist in resolving this matter urgently, as all automation testing processes have come to a halt. The website link is https://desibook.admin.linkedunion.org/. Unfortunately, I can't share the credentials as I am not permitted to do so.

### Desired behavior

After entering the Email and password and the 2FA, It should login in to the website.

### Test code to reproduce

The repo is private

### Cypress Version

13.6.3

### Node version

21.6.1

### Operating System

mac

**Comment by Jacek-fstack** [comment]
YES!
I have the same issue with OneLogin

**Comment by Hassan2419991** [comment]
It's really troublesome for me as all of my automation work stopped working all of a sudden although login functionality is working fine with the Playwright and Selenium

**Comment by jennifer-shehane** [comment]
@Hassan2419991 I can't give much guidance without a code example and being able to recreate the issue ourself. If there is not an issue related to a specific new version of Cypress (login works in previous version and not in new version), my guess would be a couple of things:

- The login page changed logic on how the login functions
- Google Chrome updated (Chrome 121 went stable yesterday) and something within their logic has changed the behavior of automation with the login. 

Could you try Chrome 120 vs Chrome 121 to see if that is the new factor?

**Comment by Hassan2419991** [comment]
I don't think it will work as I have used all the browser available in cypress runner. I have even installed chromium but still it can't let me login in to the app. The code is written in DJANGO  in python language for my the Web app.
Here is the screenshot of the browsers
<img width="1440" alt="Screenshot 2024-01-26 at 9 07 48 AM" src="https://github.com/cypress-io/cypress/assets/130425586/5b911fcb-5814-4e16-9978-cb8ef447a665">

Here is the screenshot of the Login code.
<img width="727" alt="Screenshot 2024-01-26 at 8 42 45 AM" src="https://github.com/cypress-io/cypress/assets/130425586/86b76362-559c-4a33-80dc-8bddd86c671b">
@jennifer-shehane

**Comment by Hassan2419991** [comment]
I have also tried the older versions of Cypress and I couldn't login there as-well.

**Comment by Hassan2419991** [comment]
Any updates on this? @jennifer-shehane

**Comment by jennifer-shehane** [comment]
@Hassan2419991 There's nothing actionable for us to work on at this point. Does this behavior work in a previous version of Cypress and break in a newer version of Cypress?

**Comment by Hassan2419991** [comment]
Yes this thing works in previous versions but after an update it's also not working for previous version as well. @jennifer-shehane

**Comment by Hassan2419991** [comment]
I can provide you the credential also. You can give me your email address or something like that so that I can provide you the credential. Thanks!@jennifer-shehane

**Comment by jennifer-shehane** [comment]
@Hassan2419991 You can email at support@cypress.io. It would be helpful if you narrow down the exact version that this broke in.

**Comment by Hassan2419991** [comment]
I just updated cypress to the latest version. v13.6.4 and issue is still the same. It let me login 2 or 3 time in start and after that it keep stuck on login page after entering 2FA. 
I will be sending the email and password on the provided email. You can check this scenario by simply logging in to the website from Cypress runner. You dont have to write the script, you can do it manually as-well. it will let you login 2-3 times and then it won't let you login.

**Comment by Hassan2419991** [comment]
I just updated cypress to the latest version. v13.6.4 and issue is still the same. It let me login 2 or 3 time in start and after that it keep stuck on login page after entering 2FA.
I will be sending the email and password on the provided email. You can check this scenario by simply logging in to the website from Cypress runner. You dont have to write the script, you can do it manually as-well. it will let you login 2-3 times and then it won't let you login.
@jennifer-shehane

**Comment by Hassan2419991** [comment]
Hey @jennifer-shehane  I have send the email to the provided mail. This is the subject "Cypress Github opened issue: Can't login to my Web app #28798"
I will really appreciate your help. Thanks!

**Comment by StephaneColson** [comment]
> * The login page changed logic on how the login functions
> * Google Chrome updated (Chrome 121 went stable yesterday) and something within their logic has changed the behavior of automation with the login.
> 
> Could you try Chrome 120 vs Chrome 121 to see if that is the new factor?

Hi @jennifer-shehane
I also have a kind of similar issue since a few days. 
We were using the docker image with Chrome 118 and everything was fine. Then, as I understood it in https://hub.docker.com/r/cypress/included/tags?page=1&name=13.6.3, the cypress version 13.6.3 came with Chrome 121 instead of 121 and then I get error with login.

Using parallelization, first test case logging in is ok, but the following one fails.

```
[2024-02-15T04:17:38.844Z]   Running:  Test 1         (1 of 11)                                                      

[2024-02-15T04:17:51.255Z]     0. Connexion with UI

[2024-02-15T04:18:06.385Z]       ✓ Login with UI (13969ms)

[2024-02-15T04:18:13.039Z]       ✓ 

[...]                                                                      

[2024-02-15T04:21:51.379Z]   Running:  Test 2                 (11 of 11)

[2024-02-15T04:22:01.344Z]     0. Connexion with UI

[2024-02-15T04:24:07.706Z]       (Attempt 1 of 3) Login with UI

[2024-02-15T04:26:14.071Z]       (Attempt 2 of 3) Login with UI

[2024-02-15T04:28:05.431Z]       1) Login with UI
```

I'd like to know why a [new Docker image](https://hub.docker.com/r/cypress/included/tags?page=1&name=13.6.3) of the version 13.6.3 has been done with another chrome version 121, replacing the previous one with Chrome 118. 
Is it true that when downloading cypress 13.6.3 then, depending on the time we do it, we have Chrome 118 or Chrome 121?
It was a mess to debug this knowing we changed nothing on our side (same code, same app version, same jenkins, same cypress version, etc...) :(

Thanks for your help!
